web/src/types.tsx

```diff
@@ -0,0 +1,4 @@
+export type CanvasBoardTypes = {
+  height: number,
+  width: number,
+}
```

Comment:
nit: I'm personally an advocate for naming types like this (ones that are only used as `props` for a component) as `ComponentNameProps` e.g in this case this would become `CanvasBoardProps` as it explicitly implies where this type would be used. Not a blocker though and maybe just food for thought 💭

Reply:
Another just small nit pick on this file (sorry)!

This file won't have any components within it, so shouldn't be a `tsx` file, but rather can just be `ts` file:

`.tsx` = for React components (in a React project)
`.ts` = just any other supporting Typescript, such as classes, hooks, functions, types and interfaces etc...